tests/utils/test_config.go

```diff
@@ -48,9 +47,7 @@ func CreateJfrogHomeConfig(t *testing.T, encryptPassword bool) {
 	assert.NoError(t, err)
 }
 
-func InitTestCliDetails() {
-	testApplication := cli.GetJfrogCliSecurityApp()
-
+func InitTestCliDetails(testApplication components.App) {
```

Comment:
what is the reason for the change?
There is a single place you are calling this func, where you just calling the function like this:
utils.InitTestCliDetails(cli.GetJfrogCliSecurityApp()). Why not calling it from within InitTestCliDetails function like it used to be?

Reply:
scopes and cycles